### ISSUE TITLE
Buttons' :hover, :active, and checked background-colors should be different

### ISSUE THREAD

**Issue by iBasit** [twbs/bootstrap#18177]
On v4 - http://v4-alpha.getbootstrap.com/components/buttons/#checkbox-and-radio-buttons

Checkbox button, radio button and toggle button. The state of hover is same as checked. so user cant tell if he unchecked it or is it still checked state. its bit confusing.

If you keep hover, checked and unchecked state different from each other, that will solve the issue.

![image](https://cloud.githubusercontent.com/assets/34341/10973491/951ac004-83d5-11e5-887f-43ec4729e99c.png)

**Comment by cvrebert** [comment]
I could have sworn there was an issue for this, but I can't seem to find it.

**Comment by patrickhlauke** [comment]
this sounds like something i fixed in v3...unless i'm getting confused

**Comment by patrickhlauke** [comment]
thinking of this https://github.com/twbs/bootstrap/pull/16154

**Comment by patrickhlauke** [comment]
this https://github.com/twbs/bootstrap/commit/1f153b640de55a4fff1aca3cf7331e9b7e3a408f#diff-edb9cd89b66120ef946a81de31f6bb06 should already take care of it - wondering if the v4-alpha docs simply haven't been updated yet to include this

**Comment by patrickhlauke** [comment]
confirming that once you compile and run the latest version of v4, the issue is resolved - just a case of the live v4-alpha docs being out of date

**Comment by ysds** [comment]
This problem is still confirmed now.

Demo: (dev-4 latest CSS and JS)
https://jsfiddle.net/n3f75n9x/
